tests/test_csc.py

```diff
@@ -1,42 +1,35 @@
-# import asyncio
 import unittest
 from lsst.ts import salobj
 from lsst.ts import LinearStage
 import asynctest
```

Comment:
Usually I am asked to leave a blank line between imports of standard modules and of non-standard imports which would mean that a blank line is required between these two imports. I'll leave it to you to decide whether you want to reorder the imports and introduce a blank line or not.